sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -677,12 +678,7 @@ public String getId() {
             : false;
 
         if (queryInfo != null && queryInfo.hasSelectValue()) {
```

Comment:
The condition checking here does not seem matter any more, the returning results are always the seems

I think it is because we already handled the select value much deep down in the pipeline, so no special handling is required here?

Reply:
Fixed in next iteration